libs/experimental/langchain_experimental/comprehend_moderation/amazon_comprehend_moderation.py

```diff
@@ -3,12 +3,13 @@
 from langchain.callbacks.manager import CallbackManagerForChainRun
 from langchain.chains.base import Chain
 
-from langchain_experimental.comprehend_moderation.base_moderation import (
-    BaseModeration,
-)
+from langchain_experimental.comprehend_moderation.base_moderation import BaseModeration
 from langchain_experimental.comprehend_moderation.base_moderation_callbacks import (
     BaseModerationCallbackHandler,
 )
+from langchain_experimental.comprehend_moderation.base_moderation_config import (
+    BaseModerationConfig,
+)
 from langchain_experimental.pydantic_v1 import root_validator
```

Comment:
This module is using Pydantic v1, but the Config models at the bottom are referencing v2. This won't work. Can we move everything to v1, as that's the default that LangChain is pulling in right now.

Reply:
fixed.